modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path

```diff
@@ -233,6 +233,12 @@
 	<td></td>
 </tr>
 
+<tr>
+	<td>SIDEBAR_ADVANCED_PREDEFINED_IMAGE</td>
+	<td>//div[contains(@class,'image-picker-preview')]/img[contains(@src,'${key_imageName}')]</td>
+	<td></td>
+</tr>
+
```

Comment:
@leticiachiu, the App Builder folder has been deleted from Master. Can you move this path to another file and rename it to `SIDEBAR_ADVANCED_IMAGE_PREDEFINED_VALUE`? Thanks!